content.md

```diff
@@ -10,53 +10,50 @@
 
 <div class="pagebreak"></div>
 
-
 ## 1. Installation
 
-### 1.1 From the home screen swipe down to display the notifications drawer
```

Comment:
All the instructions for enabling 'Installation from Unknown Sources' need to be added again.